src/main/java/edu/harvard/iq/dataverse/api/datadeposit/SwordServiceBean.java

```diff
@@ -150,6 +153,13 @@ public void addDatasetSubjectIfMissing(DatasetVersion datasetVersion) {
     public void setDatasetLicenseAndTermsOfUse(DatasetVersion datasetVersionToMutate, SwordEntry swordEntry) throws SwordError {
         Map<String, List<String>> dcterms = swordEntry.getDublinCore();
         List<String> listOfLicensesProvided = dcterms.get("license");
+        List<String> rights = dcterms.get("rights");
+        if (rights != null) {
```

Comment:
very small request, but I think it'd be more "readable" if this was one if statement with an &&.

Reply:
Sure, fixed in 3d4f968